source/common/http/http2/metadata_encoder.cc

```diff
@@ -127,6 +127,8 @@ std::vector<uint8_t> MetadataEncoder::payloadFrameFlagBytes() {
   return flags;
 }
 
+const std::string MetadataEncoder::payload() { return payload_.toString(); }
```

Comment:
const method like `MetadataEncoder::payload() const ` to indicate that the method doesn't change any class vars

Reply:
Changed.